src/main/java/com/ent/staySafe/staySafeController.java

```diff
@@ -7,7 +7,7 @@
 
 @Controller
 
-public class staySafeController {
```

Comment:
Oracle Java naming conventions, "Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations"
I capitalized this class name to adhere to these conventions

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html